azure-pipelines.yml

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) Microsoft Corporation.
+name: $(Date:yyyyMMdd)$(Rev:.r).0-$(SourceBranchName)
+
+pr:
+  - main
+
+trigger:
+  branches:
+    include:
+      - main
+
+jobs:
+  - job: BuildPublish
+    displayName: Build and publish
+    pool:
+      name: VSEngSS-MicroBuild2019-1ES
+    steps:
+      - checkout: none
```

Comment:
Don't you still need checkout? I don't see any other step that would pull the source code down.

Reply:
Changing.